src/features/DisplayControls/NursingTasks/components/AddEmergencyTasks.jsx

```diff
@@ -98,6 +119,79 @@ const AddEmergencyTasks = (props) => {
     }
   };
 
+  const closeModal = () => {
+    setOpenConfirmationModal(false);
+  };
+
+  const handleClose = () => {
+    if (isSaveDisabled && !atleastOneFieldFilled) {
+      updateEmergencyTasksSlider(false);
+      setShowWarningNotification(false);
+    } else {
+      setShowWarningNotification(true);
+    }
+  };
+
+  const sliderCloseActions = {
+    onCancel: () => {
+      setShowWarningNotification(false);
+      updateEmergencyTasksSlider(false);
+    },
+    onClose: () => {
+      setShowWarningNotification(false);
+    },
+  };
+
+  const createEmergencyMedicationPayload = () => {
+    const time = administrationTime.split(":");
+    const date = new Date(administrationDate);
+    date.setHours(time[0]);
+    date.setMinutes(time[1]);
+    const utcTimeEpoch = moment.utc(date).unix();
```

Comment:
can we move this method to DateTime util file, converting date to epoch?

Reply:
refactored as advised